### PR TITLE
Added GraphQL content type.

### PR DESCRIPTION
**Pull request by jaydensmith**

### CODE REVIEW

File: src/gql/types/PhoneNumberType.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ * @since 1.4.0
+ */
+class PhoneNumberType
+{
+    /**
+     * @return string
+     */
+    public static function getName(): string
+    {
+        return 'phoneNumber_PhoneNumber';
```
Comment (rynpsc) [inline src/gql/types/PhoneNumberType.php:28]: Is this the standard naming convention or would PhoneNumber be sufficient? Not too familiar with authoring GraphQL.

Comment (jaydensmith) [reply src/gql/types/PhoneNumberType.php:28]: @rynpsc generally `packageName_ModelName`

**Comment by rynpsc** [comment]
Thank you for the PR.

**Comment by rynpsc** [comment]
Phone Number 1.4.0 is now out with this PR. Thanks again.

**Pull request merged**